homeassistant/components/services.yaml

```diff
@@ -488,6 +488,54 @@ counter:
         description: Entity id of the counter to reset.
         example: 'counter.count0'
 
+timer:
+  start:
+    description: Start a timer.
+
+    fields:
+      entity_id:
+        description: Entity id of the timer to start. [optional]
+        example: 'timer.timer0'
+      weeks:
```

Comment:
Don't do it like this, instead take in a duration and use the `cv.time_period` as validator.

Reply:
That's a great validator! :)